assets/_scss/components/_forms.scss

```diff
@@ -129,3 +148,14 @@ input.usa-input-medium {
   font-style: italic;
   font-weight: normal;
 }
+
+.usa-input-buttons_inline {
+  button[type="submit"],
+  input[type="submit"] {
+    display: inline;
+  }
+  
+  .usa-button-cancel {
```

Comment:
I think this would be more robust if it used something like:

```
button:last-of-type,
input:last-of-type {
```

That would mean the user could put the buttons in any order they please and wouldn't be confined to needing a `usa-button-cancel`